test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/JSExportStaticTest.scala

```diff
@@ -182,4 +256,32 @@ object JSExportStaticTest {
     @JSExportStatic
     def alsoExistsAsMember(x: Int): Int = x * 5
   }
+
+  @ScalaJSDefined
+  class StaticExportFields extends js.Object {
+    val alsoExistsAsMember: Int = 5
```

Comment:
Did you want to introduce a static field with this name as well? (and maybe a test?)